backend/src/main/kotlin/app/ehrenamtskarte/backend/common/webservice/GraphQLContext.kt

```diff
@@ -16,10 +16,15 @@ data class GraphQLContext(
     val backendConfiguration: BackendConfiguration,
     val regionIdentifierByPostalCode: List<Pair<String, String>>,
     val request: HttpServletRequest,
-) : GraphQLContext {
```

Comment:
:thought_balloon: maybe we could also rename our GraphQLContext to avoid collisions with graphql.GraphQLContext

CustomGraphQLContext ?

Reply:
I didn't want to let this PR grow even more, so I kept the name. But I think you're right, the name clashes with a library class and the `dfe.graphQlContext.context` accessor chain also does not look very good IMHO. 

Our `GraphQLContext` looks like a kitchen sink with all kinds of objects related to the request, and there is also the `fun GraphQLContext.getAdministrator()` extension function, which could be included directly in the class.
 
I would refactor this in a later PR, when I'm more familiar with the code base.